server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/AbstractRedirect.java

```diff
@@ -1510,24 +1511,29 @@ public static void changeSubject(MimeMessage message, String newValue) throws Me
      *                header was not present (in this case it always return null).
      * @return the MIME charset name or null if no encoding applied
      */
-    static private String determineMailHeaderEncodingCharset(String rawText) {
-        if (rawText == null)
+    @VisibleForTesting String determineMailHeaderEncodingCharset(String rawText) {
```

Comment:
oh, I actually deleted the code because I solved the problem with another solution. Don't bother with this comment.